File: tests/php/testcases/AddSimpleTest.php
<?
require_once 'base.php';

class AddSimpleTestCase extends TestServiceTestCase {

    public function test_null() {
        $this->assertEquals($this->service->add_simple(null, null), '');
    }

    public function test_empty() {
        $this->assertEquals($this->service->add_simple('', ''), '');
    }

    public function test() {
        $this->assertEquals(
            $this->service->add_simple('hello', ' world'), 'hello world');
    }
}

File: tests/mono/testcases/AddSimpleTest.cs
using System;
using NUnit.Framework;

namespace TestCases {

    [TestFixture()]
    public class AddSimpleTest : TestServiceTest {

        [Test()]
        public void test_null() {
            Assert.AreEqual("", service.add_simple(null, null));
        }

        [Test()]
        public void test_empty() {
            Assert.AreEqual("", service.add_simple("", ""));
        }

        [Test()]
        public void test() {
            Assert.AreEqual(
                "hello world", service.add_simple("hello ", "world"));
        }
    }
}
